### PR TITLE
Disable all translations except English for Beta. Closes #4398

### LINKED ISSUE #4398: Disable all translations except English for Beta
### User Story

As a user I want to have good user experience when using the app , so UI should not be glitched or overlapped while being not properly localized for my language

### Description

*Type*: Bug

*Summary*: Disable all translations except English for Beta so we have good user experience

#### Expected behavior
Status UI is English only

#### Actual behavior
Status UI is localized when system language is changed, but this needs to be redone . As result we have a lot of glitches and overlaps which is not nice

### PR DESCRIPTION
**Pull request by pedropombeiro**
fixes #4398 

### Steps to test:
- Set phone language to French
- Open Status
- App should show all text in English

status: ready

### CODE REVIEW

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/gyAqNU for branch PR-4429

<blockquote><div><strong><a href="https://i.diawi.com/gyAqNU">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/mYc5Vv for branch PR-4429

<blockquote><div><strong><a href="https://i.diawi.com/mYc5Vv">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/null for branch PR-4429

<blockquote><div><strong><a href="https://i.diawi.com/null">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/XyoNTg for branch PR-4429

<blockquote><div><strong><a href="https://i.diawi.com/XyoNTg">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

File: src/status_im/i18n.cljs [hunk 1]
```diff
@@ -65,65 +65,66 @@
...
-                             :zh-hans-mo  zh-hans/translations
-                             :zh-hant     zh-hant/translations
-                             :zh-hant-tw  zh-hant-tw/translations
-                             :zh-hant-sg  zh-hant-sg/translations
-                             :zh-hant-hk  zh-hant-hk/translations
-                             :zh-hant-cn  zh-hant/translations
-                             :zh-hant-mo  zh-hant/translations
-                             :zh-wuu      zh-wuu/translations
-                             :zh-yue      zh-yue/translations})
+(def translations-by-locale {:en          en/translations}) ; Temporarily disable all languages except English
```
Comment (jeluard) [inline src/status_im/i18n.cljs]: This style of comments is pretty intrusive. Please rather use `#_` before `translations-by-locale` form.

**Review by jeluard — CHANGES_REQUESTED** [review]

File: src/status_im/i18n.cljs (hunk 1 above)
Comment (jeluard) [inline src/status_im/i18n.cljs]: We usually add comments before the commented form.

Comment (pedropombeiro) [reply src/status_im/i18n.cljs]: Will do, thanks for the tips!

Comment (pedropombeiro) [reply src/status_im/i18n.cljs]: Fixed

**Review by jeluard — APPROVED** [review]

**Review by goranjovic — APPROVED** [review]

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/5w46N8 for branch PR-4429

<blockquote><div><strong><a href="https://i.diawi.com/5w46N8">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/4qpKkH for branch PR-4429

<blockquote><div><strong><a href="https://i.diawi.com/4qpKkH">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/koBz5c for branch PR-4429

<blockquote><div><strong><a href="https://i.diawi.com/koBz5c">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/nyU7Qr for branch PR-4429

<blockquote><div><strong><a href="https://i.diawi.com/nyU7Qr">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Pull request merged**